scripts/rv_expression_association/subset_variants.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# pylint: disable=no-value-for-parameter,missing-function-docstring,missing-module-docstring
+
+import click
+import hail as hl
+from cpg_utils.hail_batch import dataset_path, init_batch
+
+
+@click.command()
+@click.option('--input-mt-path', required=True)  # 'mt/v7.mt'
+@click.option('--genes', required=True)  # 'LMNA'
+@click.option(
+    '--output-mt-path', required=True
+)  # 'gs://cpg-tob-wgs-main-analysis/tob_wgs_rv/pseudobulk_rv_association/significant_genes_burden_max.mt'
+def subset_variants(
+    input_mt_path: str,
+    genes: str,
+    output_mt_path: str,
+):
+    init_batch()
+
+    # read ht objects to extract variant names
+    variants = []
+    genes_of_interest = genes.split(' ')
```

Comment:
Have you tested this somewhere? I don't know how well click is going to handle whitespace-separated arguments once they're passed via analysis-runner

Reply:
So I think this should work as it did for the hail batch workflow? (https://github.com/populationgenomics/cellregmap-pipeline/blob/main/batch.py#L704)

And no I tried to test but I don't have access to the main-analysis bucket from main I think! Is the best approach to use [your script](https://github.com/populationgenomics/analysis-runner/blob/main/scripts/subset_hail_table.py) to copy a couple of those summary hts to `test` so I can test this?